internal/integration/reql_tests/reql_datum_binary_test.go

```diff
@@ -56,7 +56,7 @@ func (suite *DatumBinarySuite) TearDownSuite() {
 }
 
 func (suite *DatumBinarySuite) TestCases() {
-	suite.T().Log("Running DatumBinarySuite: Tests of converstion to and from the RQL binary type")
+	suite.T().Log("Running DatumBinarySuite: Tests of conversation to and from the RQL binary type")
```

Comment:
This should be conversion, not conversation.

```suggestion
	suite.T().Log("Running DatumBinarySuite: Tests of conversion to and from the RQL binary type")
```

Reply:
done